source/run-command/tests/unified/run-command.yml

```diff
@@ -0,0 +1,242 @@
+description: runCommand spec requirements
+
+schemaVersion: "1.5"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents: [commandStartedEvent]
+  - database:
+      id: &plainDb plainDb
+      client: *client
+      databaseName: *plainDb
+  - collection:
+      id: &collection collection
+      database: *plainDb
+      collectionName: *collection
+  - database:
+      id: &dbWithRC dbWithRC
+      client: *client
+      databaseName: *dbWithRC
+      databaseOptions:
+        readConcern:
+          level: 'local'
+  - database:
+      id: &dbWithWC dbWithWC
+      client: *client
+      databaseName: *dbWithWC
+      databaseOptions:
+        writeConcern:
+          w: 0
+  - session:
+      id: &session session
+      client: *client
+  # Stable API test
+  - client:
+      id: &stableApiClient stableApiClient
+      observeEvents: [commandStartedEvent]
+      serverApi:
+        version: "1"
+        strict: true
+  - database:
+      id: &stableApiDb stableApiDb
+      client: *stableApiClient
+      databaseName: *stableApiDb
+
+initialData:
+- collectionName: *collection
+  databaseName: *plainDb
+  documents: [ { _id: 0 } ]
+
+tests:
+  - description: always attaches $db to given command
+    operations:
+      - name: runCommand
+        object: *plainDb
+        arguments:
+          commandName: ping
+          command:
+            ping: 1
+        expectResult:
+          ok: 1
+    expectEvents:
+      - client: *client
+        events:
+        - commandStartedEvent:
+              command:
+                ping: 1
+                $db: *plainDb
+              commandName: ping
+
+  - description: attaches the provided session lsid to given command
+    operations:
+      - name: runCommand
+        object: *plainDb
+        arguments:
+          commandName: ping
+          command:
+            ping: 1
+          session: *session
+        expectResult:
+          ok: 1
+    expectEvents:
+      - client: *client
+        events:
+        - commandStartedEvent:
+              command:
+                ping: 1
+                lsid: { $$sessionLsid: *session }
+                $db: *plainDb
+              commandName: ping
+
+  - description: attaches the provided $readPreference to given command
+    operations:
+      - name: runCommand
+        object: *plainDb
+        arguments:
+          commandName: ping
+          command:
+            ping: 1
+          readPreference: 'nearest'
```

Comment:
Yes it does, it will invoke our runCommand like so:
```ts
await db.commmand({ ping: 1 }, { readPreference: 'nearest' })
```
Nesting `'nearest'` inside of a `mode` property also works.